pkg/storaged/client.js

```diff
@@ -1030,7 +1030,7 @@ function stratis2_start() {
                             .input(passphrase);
                 };
 
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18679-20230425-100703-453867ac-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)